### ISSUE TITLE
3 projects that use SignAssembly, but not PublicSign

### ISSUE THREAD

**Issue by omajid** [dotnet/source-build#2907]
I found 3 locations in source-build-externals (in 6.0) that use `<SignAssembly>true</SignAssembly>` without `<PublicSign>true</PublicSign>`.

To get source-build working in an environment with RSA+SHA1 disabled, I had to do this hack to source-build:

```sh
sed -i -E 's|<SignAssembly>true</SignAssembly>|<SignAssembly>true</SignAssembly><PublicSign>true</PublicSign>|' \
  src/source-build/src/MSBuildLocator/src/MSBuildLocator/Microsoft.Build.Locator.csproj

sed -i -E 's|<SignAssembly>true</SignAssembly>|<SignAssembly>true</SignAssembly><PublicSign>true</PublicSign>|' \
  src/source-build/src/humanizer/src/Humanizer/Humanizer.csproj

sed -i -E 's|<SignAssembly>true</SignAssembly>|<SignAssembly>true</SignAssembly><PublicSign>true</PublicSign>|' \
  src/source-build/src/cssparser/Directory.Build.props
```

**Comment by crummel** [comment]
Fixed with https://github.com/dotnet/source-build-reference-packages/pull/566.

**Comment by MichaelSimons** [comment]
I don't think this is fixed yet.  There are more repos than SBRP that require changes.  Specifically https://github.com/dotnet/source-build-externals/pull/99 is needed.

**Comment by omajid** [comment]
There are a few additional projects that need more changes (including dotnet/razor, at a minimum).

Do we want to use this issue to track all of those? Or should we keep this issue focused on source-build-externals?

**Comment by MichaelSimons** [comment]
>Do we want to use this issue to track all of those? Or should we keep this issue focused on source-build-externals?

I would prefer to have repo specific issue logged for any repo level changes need to support this.  The value in this is to provide visibility to the repo owners.
